source/features/sticky-discussion-sidebar.css

```diff
@@ -2,7 +2,7 @@
 Set the sidebar container's height to make it fit its contents
 (instead of being the full height of the discussion)
 */
-.discussion-sidebar {
+#discussion_bucket > :last-child {
```

Comment:
This points at the same element as the JavaScript file does. Since there is no way to get the parent element in CSS, I used this.

Reply:
I think this was the layout change that made them remove the class that existed before.

This works for now, but we need to mention what the selector targets, preferably in the comments above the selector.

Reply:
It does say it targets the "sidebar container" above it, if it's not good enough I'll add a sentence similar to my comment above.

Reply:
That comment does make sense, but I was referring to maybe adding a comment on why that specific CSS selector was used. Doesn't make much of a difference though, but I thought it would a little bit more clear.

Reply:
Eh, it doesn't hurt to explain a little more. Thanks you :)